encoder/vaapiencoder_h264.cpp

```diff
@@ -1319,9 +1321,10 @@ bool VaapiEncoderH264::addSliceHeaders (const PicturePtr& picture) const
         sliceParam->slice_qp_delta = initQP() - minQP();
         if (sliceParam->slice_qp_delta > 4)
             sliceParam->slice_qp_delta = 4;
-        sliceParam->slice_alpha_c0_offset_div2 = 2;
-        sliceParam->slice_beta_offset_div2 = 2;
 
+        sliceParam->disable_deblocking_filter_idc = !m_videoParamAVC.enableDeblockFilter;
+        sliceParam->slice_alpha_c0_offset_div2 = m_videoParamAVC.deblockAlphaOffset;
```

Comment:
Hi @xuguangxin : 
1. Yeah, I also noted it. I didn't raise it because I think it's ok to set it to be zero. It has a little effect to deblocking stress,   but has no necessary relation on encoding quality. My concern is it may effect QA test result. So would better change it to 2.
2. Good catch. Here deblockAlphaOffset should same as FilterOffsetA defined in h264 spec. Should change to sliceParam->slice_alpha_c0_offset_div2 = m_videoParamAVC.deblockAlphaOffset / 2. 
Same to  slice_beta_offset_div2 and deblockBetaOffset.